app/Http/Controllers/Users/LDAPImportController.php

```diff
@@ -6,45 +6,66 @@
 use Illuminate\Http\Request;
 use App\Http\Controllers\Controller;
 use Illuminate\Support\Facades\Artisan;
+use App\Models\LdapAd;
 
 class LDAPImportController extends Controller
 {
+    /**
+     * An Ldap instance.
+     *
+     * @var LdapAd
+     */
+    protected $ldap;
+
+    /**
+     * __construct.
+     *
+     * @param LdapAd $ldap
+     */
+    public function __construct(LdapAd $ldap)
+    {
+        $this->ldap = $ldap;
+    }
 
     /**
-     * Return view for LDAP import
+     * Return view for LDAP import.
      *
      * @author Aladin Alaily
-     * @since [v1.8]
+     * @author Wes Hulette <jwhulette@gmail.com>
+     *
+     * @since 5.0.0
+     *
      * @return \Illuminate\Contracts\View\View
+     *
      * @throws \Illuminate\Auth\Access\AuthorizationException
      */
     public function create()
     {
         $this->authorize('update', User::class);
         try {
-            $ldapconn = Ldap::connectToLdap();
-            Ldap::bindAdminToLdap($ldapconn);
-
+            $this->ldap->testLdapAdUserConnection();
         } catch (\Exception $e) {
             return redirect()->route('users.index')->with('error', $e->getMessage());
         }
 
         return view('users/ldap');
     }
 
-
     /**
      * LDAP form processing.
      *
      * @author Aladin Alaily
-     * @since [v1.8]
+     * @author Wes Hulette <jwhulette@gmail.com>
+     *
+     * @since 5.0.0
+     *
      * @return \Illuminate\Http\RedirectResponse
      */
     public function store(Request $request)
     {
         // Call Artisan LDAP import command.
         $location_id = $request->input('location_id');
-        Artisan::call('snipeit:ldap-sync', ['--location_id' => $location_id, '--json_summary' => true]);
+        Artisan::call('snipeit:ldapAd-sync', ['--location_id' => $location_id, '--json_summary' => true]);
```

Comment:
I could see an argument to be made for keeping the old artisan command - then documentation doesn't need to change. Especially if the command isn't really any different.